persistence/src/test/java/edu/unc/lib/dl/persist/services/edit/ExpireEmbargoServiceIT.java

```diff
@@ -0,0 +1,275 @@
+        verify(operationsMessageSender).sendOperationMessage(
+                eq(SoftwareAgent.embargoExpirationService.getFullname()),
+                eq(JMSMessageUtil.CDRActions.EDIT_ACCESS_CONTROL),
+                pidListCaptor.capture());
+        assertTrue(pidListCaptor.getValue().contains(pid));
+    }
+
+
+    @Test
+    public void expireMultipleEmbargoesTest() throws Exception {
+        Calendar embargoUntil = yesterday();
+        // create first embargoed collection
+        createCollectionInUnit(new AclModelBuilder("Collection with embargo")
+                .addEmbargoUntil(embargoUntil)
+                .model);
+        PID pid1 = collObj1.getPid();
+        // create second embargoed collection
+        createSecondCollectionInUnit(new AclModelBuilder("Another collection with embargo")
+                .addEmbargoUntil(embargoUntil)
+                .model);
+        PID pid2 = collObj2.getPid();
+
+        TestHelper.setContentBase(baseAddress);
+        treeIndexer.indexAll(baseAddress);
+
+        service.expireEmbargoes();
+
+        RepositoryObject target1 = repoObjLoader.getRepositoryObject(pid1);
+        RepositoryObject target2 = repoObjLoader.getRepositoryObject(pid2);
+        assertNoEmbargo(target1);
+        assertNoEmbargo(target2);
+
+        List<String> eventDetails1 = getEventDetails(target1);
+        List<String> eventDetails2 = getEventDetails(target2);
+        assertEquals(1, eventDetails1.size());
+        assertEquals(1, eventDetails2.size());
+
+        assertEventWithDetail(eventDetails1, "Expired an embargo for " + pid1.toString() + " which ended " + formatDateToUTC(embargoUntil.getTime()));
+        assertEventWithDetail(eventDetails2, "Expired an embargo for " + pid2.toString() + " which ended " + formatDateToUTC(embargoUntil.getTime()));
+
+        verify(operationsMessageSender).sendOperationMessage(
+                eq(SoftwareAgent.embargoExpirationService.getFullname()),
+                eq(JMSMessageUtil.CDRActions.EDIT_ACCESS_CONTROL),
+                pidListCaptor.capture());
+        List<PID> pids = pidListCaptor.getValue();
+        assertTrue(pids.contains(pid1));
+        assertTrue(pids.contains(pid2));
+    }
+
+    @Test
+    public void expireNoEmbargoesTest() throws Exception {
+        createCollectionInUnit(null);
+        PID pid = collObj1.getPid();
+        TestHelper.setContentBase(baseAddress);
+        treeIndexer.indexAll(baseAddress);
+
+        service.expireEmbargoes();
+
+        // collection was not created with an embargo and should not have one
+        RepositoryObject target = repoObjLoader.getRepositoryObject(pid);
+        assertNoEmbargo(target);
+
+        // no event should have been created since no embargoes were expired
+        List<String> eventDetails = getEventDetails(target);
+        assertEquals(0, eventDetails.size());
```

Comment:
Check that no messages were sent. You can use the `never()` method with `verify` for that